### PR TITLE
Introduce mechanism to avoid loading default config files

### PR DESCRIPTION
**Pull request by guillermocalvo**

### CODE REVIEW

**Comment by sdelamo** [comment]
@guillermocalvo check [AWS distributed configuration](https://micronaut-projects.github.io/micronaut-aws/latest/guide/#distributedconfiguration). Can we implement a similar feature or paradigm in AWS?. I would like to have the same developer experience in both modules.

**Review by sdelamo — CHANGES_REQUESTED** [review]
I would like the following changes #934 #933

File: gcp-secret-manager/src/main/java/io/micronaut/gcp/secretmanager/SecretManagerConfigurationClient.java [hunk 1]
```diff
@@ -22,7 +22,6 @@
 import java.util.Objects;
 import java.util.Optional;
```
Comment (guillermocalvo) [inline gcp-secret-manager/src/main/java/io/micronaut/gcp/secretmanager/SecretManagerConfigurationClient.java:23]:
```suggestion
```

**Review by sdelamo — APPROVED** [review]

**Comment by guillermocalvo** [comment]
> Can we implement a similar feature or paradigm in AWS?. I would like to have the same developer experience in both modules.

@sdelamo https://github.com/micronaut-projects/micronaut-aws/pull/1886.

**Pull request merged**